commands/audit/sca/python/python.go

```diff
@@ -103,9 +115,63 @@ func getDependencies(auditPython *AuditPython) (dependenciesGraph map[string][]s
 		sca.LogExecutableVersion("python")
 		sca.LogExecutableVersion(string(auditPython.Tool))
 	}
+	if auditPython.IsCurationCmd {
+		pipUrls, errProcessed := processPipDownloadsUrlsFromReportFile()
+		if errProcessed != nil {
+			err = errProcessed
+			return
+		}
+		params.SetDownloadUrls(pipUrls)
+	}
```

Comment:
```suggestion
	if !auditPython.IsCurationCmd {
        return
	}
	if pipUrls, errProcessed := processPipDownloadsUrlsFromReportFile(); errProcessed != nil {
		err = errProcessed
	} else {
        params.SetDownloadUrls(pipUrls)
    }
```
keep the code flat for better readability.

Reply:
Fixed